apps/gitness/src/pages/repo/repo-branch-list.tsx

```diff
@@ -59,15 +95,26 @@ export function ReposBranchesListPage() {
       )
     }
 
+    //get the data arr from behindAhead
+    const behindAhead =
+      getBranchDivergence?.map(divergence => {
+        return {
+          behind: divergence.behind,
+          ahead: divergence.ahead
+        }
+      }) || []
+
     return (
       <BranchesList
-        branches={brancheslistData?.map((branch: TypesBranch) => {
+        branches={brancheslistData?.map((branch: TypesBranch, index) => {
+          const { ahead: branchAhead, behind: branchBehind } = behindAhead[index] || {}
           return {
-            name: branch.name,
-            sha: branch.commit?.sha,
+            id: index,
+            name: branch.name || '',
+            sha: branch.commit?.sha || '',
             timestamp: timeAgo(branch.commit?.committer?.when || ''),
             user: {
-              name: branch.commit?.committer?.identity?.name,
+              name: branch.commit?.committer?.identity?.name || 'Unknown User',
```

Comment:
Let's remove this placeholder name `Unknown User`. We'll add a proper placeholder later on. It's ok if it renders empty for now.